### PR TITLE
Updated README file for smart contracts docs build

### PR DESCRIPTION
**Pull request by wonderwomancode**
All in the title

### CODE REVIEW

File: packages/contracts/bin/build_docs.sh [hunk 1]
```diff
@@ -36,10 +36,105 @@ sed -i -e "s/IPublicLockV${latest_template}/PublicLock (v${latest_template})/g"
...
+# Interface References
+
+This section of the docs is generated automatically from the smart contract 
+interface references. 
+<a href="https://github.com/unlock-protocol/unlock/tree/master/smart-contracts/contracts/interfaces">
+    <button class="button button-primary">See them on GitHub</button>
+</a>
+
+## Using Smart Contract Interfaces
+A Solidity contract interface is a list of function definitions without
```
Comment (julien51) [inline packages/contracts/bin/build_docs.sh:57]:
```suggestion

A Solidity contract interface is a list of function definitions without 
```

File: packages/contracts/bin/build_docs.sh [hunk 2]
```diff
@@ -36,10 +36,105 @@ sed -i -e "s/IPublicLockV${latest_template}/PublicLock (v${latest_template})/g"
...
+    <button class="button button-primary">See them on GitHub</button>
+</a>
+
+## Using Smart Contract Interfaces
+A Solidity contract interface is a list of function definitions without 
+implementation. This allows for a seperation between the interface and the 
+implementation much like Abstract Base Classes in Python or C++. 
+
+You can use these interfaces in your own smart contracts to interact with
+Unlock Protocol smart contracts, however they cannot be used to instaiate a
```
Comment (julien51) [inline packages/contracts/bin/build_docs.sh:62]:
```suggestion
Unlock Protocol smart contracts, however, they cannot be used to instantiate a
```

File: packages/contracts/bin/build_docs.sh [hunk 3]
```diff
@@ -36,10 +36,105 @@ sed -i -e "s/IPublicLockV${latest_template}/PublicLock (v${latest_template})/g"
...
+
+### npm module
+
+We\'ve packaged the interfaces along with the contracts. 
+<a href="https://www.npmjs.com/package/@unlock-protocol/contracts">
+    <button class="button button-primary">See it on npm</button>
+</a>
+
+### Load it in your project
+Add them to your project using yarn
```
Comment (julien51) [inline packages/contracts/bin/build_docs.sh:73]:
```suggestion

Add them to your project using yarn
```

File: packages/contracts/bin/build_docs.sh [hunk 4]
```diff
@@ -36,10 +36,105 @@ sed -i -e "s/IPublicLockV${latest_template}/PublicLock (v${latest_template})/g"
...
+```
+or npm
+```shell
+npm i @unlock-protocol/contracts
+```
+
+### Examples use cases
+
+#### Creating Hooks
+Let\'s say for instance you\'d like people to be able to sell their memberships,
```
Comment (julien51) [inline packages/contracts/bin/build_docs.sh:85]:
```suggestion

Let\'s say for instance you\'d like people to be able to sell their memberships,
```

**Review by julien51 — APPROVED** [review]

**Review by clemsos — APPROVED** [review]
LG 👍

**Comment by julien51** [comment]
Hum. I think this broke master. Reverting.

**Pull request merged**